news/_posts/2021-05-31-prestashop-1-7-8-0-beta-release.md

```diff
@@ -0,0 +1,247 @@
+- A new order internal note is available in the Order View Page [#14753](https://github.com/PrestaShop/PrestaShop/issues/14753)
+- Back Office users can upload custom avatars [#18653](https://github.com/PrestaShop/PrestaShop/issues/18653)
+
+### Notable fixes
+
+### **Topwatchers:**
+
+- When a product had a specific price applied and was discounted by unit, the discount displayed on the product page was false [#16163](https://github.com/PrestaShop/PrestaShop/issues/16163) (Major)
+- When Clear cache button was clicked twice, the user would see an error page [#11105](https://github.com/PrestaShop/PrestaShop/issues/11105) (Major)
+- Profiler was not compatible with hooks and module [#9659](https://github.com/PrestaShop/PrestaShop/issues/9659)
+- Double click on "Add to cart" on the product page FO was freezing the page [#9634](https://github.com/PrestaShop/PrestaShop/issues/9634) (Major)
+
+### Major bugs:
+
+**Front-Office**
+
+- Guest Order Tracking redirected to shop Homepage when Friendly URLs were disabled [#20194](https://github.com/PrestaShop/PrestaShop/issues/20194)
+- An error was displayed on checkout when Legal compliance module was installed [#12509](https://github.com/PrestaShop/PrestaShop/issues/12509)
+- Checkout slowdown when many orders were made with a Guest account [#16584](https://github.com/PrestaShop/PrestaShop/issues/16584)
+- Ecotax was displayed tax excl. instead of tax incl. [#18835](https://github.com/PrestaShop/PrestaShop/issues/18835)
+- Confirmation button on Checkout should be disabled if Terms and conditions are not approved [#19161](https://github.com/PrestaShop/PrestaShop/issues/19161)
+
+**Back-Office**
+
+- An exception was thrown when adding an invalid parameter to the Language form and "No picture" image [#22500](https://github.com/PrestaShop/PrestaShop/issues/22500)
+- In Back-Office, some of the JavaScript code relied on form field's ids and attributes that were modified in 1.7.8 [#21819](https://github.com/PrestaShop/PrestaShop/issues/21819)
+- When creating a customer account, the number of characters was only checked browser-side [#19505](https://github.com/PrestaShop/PrestaShop/issues/19505)
+- It was possible to create a catalog price rule with percentage value greater than 100% [#19013](https://github.com/PrestaShop/PrestaShop/issues/19013)
+- A bug in TinyMCE Update 4.0.16 was creating errors, fixed by upgrading TinyMCE to 4.9.8 [#11011](https://github.com/PrestaShop/PrestaShop/issues/11011)
+- Unit price was reset to 0 upon activating/disabling or duplicating products from the Product List [#10792](https://github.com/PrestaShop/PrestaShop/issues/10792)
+- Installing PrestaShop with SSL was complex because some settings were not acknowledged [#10482](https://github.com/PrestaShop/PrestaShop/issues/10482)
+
+**WebServices**
+
+- The id_group_shop was shared instead of id_shop_group in webservice parameter [#19566](https://github.com/PrestaShop/PrestaShop/issues/19566)
+
+**CO**
```

Comment:
What means "CO"?

I think it means Core. But merchant does not know what it means.

Can I suggest `Install` or `Distribution` or `Release`?
```suggestion
**Distribution**
```